docs/pipelines/how-it-works/index.md

```diff
@@ -8,9 +8,9 @@
 
 Gruntwork Pipelines can be used with any external CI/CD tool. The role of the CI/CD tool is to trigger jobs
 inside Gruntwork Pipelines. We have [example configurations](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/examples/for-production/infrastructure-live/_ci/scripts)
-that identify changed terraform modules and call the Gruntwork Pipelines invoker lambda.
+that identify changed terraform modules and call the Gruntwork Pipelines invoker Lambda function.
```

Comment:
I think some of these changes are getting picked up since `regenerate` wasn't run on previous branches after PR feedback was incorporated into `_docs-sources`